build/configs/artik053/Make.defs

```diff
@@ -65,6 +71,9 @@ ifeq ($(WINTOOL),y)
   ARCHINCLUDES = -I. -isystem "${shell cygpath -w $(TOPDIR)/include}" -isystem "${shell cygpath -w $(TOPDIR)/../framework/include}" -isystem "${shell cygpath -w $(TOPDIR)/../external/include}"
   ARCHINCLUDES += -isystem "${shell cygpath -w $(TOPDIR)/net/lwip/src/include}"
   ARCHXXINCLUDES = -I. -isystem "${shell cygpath -w $(TOPDIR)/include}" -isystem "${shell cygpath -w $(TOPDIR)/../external/include}"
+ifeq ($(CONFIG_HAVE_CXX),y)
+ARCHXXINCLUDES = -I. -isystem "${shell cygpath -w $(TOPDIR)/include}" -isystem "${shell cygpath -w $(TOPDIR)/../framework/include}" -isystem "${shell cygpath -w $(TOPDIR)/../external/include}" -isystem "${shell cygpath -w $(TOPDIR)/../external/include/libcxx}"
```

Comment:
I don't agree redefinition because it causes duplicated change. When we need to modify, we should modify several places for the same change.
You did
```
AA = BB CC DD
ifeq (config_cxx, y)
AA = BB CC DD EE
endif
```
I suggest
```
AA = BB CC DD
ifeq (config_cxx, y)
AA += EE
endif
```

This is what I left before.

Reply:
same as all other lines.